src/macaron/__main__.py

```diff
@@ -365,6 +379,13 @@ def main(argv: list[str] | None = None) -> None:
         help=("The path to provenance expectation file or directory."),
     )
 
+    single_analyze_parser.add_argument(
+        "-pf",
+        "--provenance-file",
+        required=False,
+        help=("The path to provenance document file."),
```

Comment:
```suggestion
        help=("The path to the provenance file in in-toto format."),
```

Reply:
Fixed in https://github.com/oracle/macaron/pull/654/commits/b32d59d9d487029bf47eae9154d678cd052726fc